CMakeLists.txt

```diff
@@ -1,12 +1,23 @@
 CMAKE_MINIMUM_REQUIRED(VERSION 3.11)
 
+SET(CMAKE_PROJECT_INCLUDE_BEFORE
+	"${CMAKE_SOURCE_DIR}/cmake/IncludeBeforeProject.cmake"
+)
+
+PROJECT(gemrb)
+
 SET(CMAKE_CXX_STANDARD 14)
 
 LIST(APPEND CMAKE_MODULE_PATH
-	"${CMAKE_SOURCE_DIR}/cmake"
+	"${CMAKE_CURRENT_SOURCE_DIR}/cmake"
 	"${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/"
 )
 
+IF(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
```

Comment:
it would be cleaner to test APPLE here, since that's what we use everywhere else.